rust/stracciatella/src/schemas/yaml/explosives.schema.yaml

```diff
@@ -0,0 +1,71 @@
+$schema: http://json-schema.org/schema#
+type: array
+title: Explosives
+description: |
+  **Limitations**: Some explosives are still referenced in source code, so they should not be changed.
```

Comment:
The itemIndex or something else?

Reply:
Yeah, the item indexes are used for some things, e.g. explosive weapons.

Reply:
It would be good to explain this a bit more. As a modder I wouldn't be sure what I can and cannot modify. Are some of the explosives off-limits, are just some fields problematic, that sort of thing.

Reply:
Right now I just stated that this should not be adapted. I will update this in the future.